pkg/cmd/version/version.go

```diff
@@ -114,7 +114,11 @@ func Command(p cli.Params) *cobra.Command {
 					fmt.Fprintf(cmd.OutOrStdout(), "Invalid component value\n")
 				}
 			} else {
-				fmt.Fprintf(cmd.OutOrStdout(), "Client version: %s\n", clientVersion)
+				if component == "" {
+					fmt.Fprintf(cmd.OutOrStdout(), "Client version: %s\n", clientVersion)
+				} else {
+					fmt.Fprintf(cmd.OutOrStdout(), "%s\n", clientVersion)
```

Comment:
I don't think this is right though, because as is, `tkn version --component=pipeline` with a error connecting kubernetes, it will print the client version without an error, which doesn't make sense to me. 

We should print the client version **if** the component is `client`, otherwise we should either error out or print `unknown`.

Reply:
Yes, my bad.
I will correct it.